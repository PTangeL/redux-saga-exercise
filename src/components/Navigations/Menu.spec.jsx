import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import Menu from './Menu'

describe('<Menu />', () => {
  const props = {

    isOpen: false,
    pageWrapId: '',
    toggleMenu: () => {}
  }
  it('should render component with props', () => {
    const wrapper = shallow(<Menu {...props} />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
