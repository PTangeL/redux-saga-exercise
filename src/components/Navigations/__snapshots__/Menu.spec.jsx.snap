// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Menu /> should render component with props 1`] = `
<SideMenu
  isOpen={false}
  toggleMenu={[Function]}
>
  <h3
    style={
      Object {
        "paddingTop": "50px",
      }
    }
  >
    Beers I like:
  </h3>
  <hr />
  <p>
    This is exercise question 5 ;)
  </p>
</SideMenu>
`;
